Source/Internal/Models/VirtusizeGetSizeParams.swift

```diff
@@ -35,13 +35,22 @@ internal struct VirtusizeGetSizeParams: Codable {
     var productType: String = ""
 	/// The user's gender
     var userGender: String = ""
+	/// The user's height
+	var userHeight: String = ""
+	/// The user's weight
+	var userWeight: String = ""
+	/// The external product ID provided by the client
+	var externalProductId: String = ""
 
     private enum CodingKeys: String, CodingKey {
```

Comment:
The API accepts camelCase as well

```sh
curl 'https://services.virtusize.com/ds-functions/size-rec/get-size-new' \
-X 'POST' \
-H 'Content-Type: application/json;charset=UTF-8' \
-H 'Pragma: no-cache' \
-H 'Accept: */*' \
-H 'Accept-Language: en-us' \
-H 'Accept-Encoding: gzip, deflate, br' \
-H 'Cache-Control: no-cache' \
-H 'Host: services.virtusize.com' \
-H 'Origin: http://localhost:1234' \
-H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.1 Safari/605.1.15' \
-H 'Connection: keep-alive' \
-H 'Referer: http://localhost:1234/' \
-H 'Content-Length: 1277' \
--data-binary '{"additionalInfo":{"brand":"","fit":"regular","sizes":{},"gender":null,"modelInfo":{}},"bodyData":{"armpitHeight":{"value":1295,"predicted":true},"bicep":{"value":146.5,"predicted":true},"bust":{"value":442,"predicted":true},"bustWidth":{"value":296,"predicted":true},"headHeight":{"value":241,"predicted":true},"hip":{"value":469.5,"predicted":true},"hipHeight":{"value":871,"predicted":true},"hipWidth":{"value":325,"predicted":true},"inseam":{"value":809,"predicted":true},"kneeHeight":{"value":460,"predicted":true},"neck":{"value":367,"predicted":true},"rise":{"value":260,"predicted":true},"shoulder":{"value":435,"predicted":true},"shoulderHeight":{"value":1421,"predicted":true},"shoulderWidth":{"value":398,"predicted":true},"sleeve":{"value":834,"predicted":true},"sleeveLength":{"value":598,"predicted":true},"thigh":{"value":279.5,"predicted":true},"waist":{"value":381.5,"predicted":true},"waistHeight":{"value":1065,"predicted":true},"waistWidth":{"value":267,"predicted":true}},"itemSizesOrig":{"M":{"height":780,"shoulder":590,"bust":670,"sleeve":861},"L":{"height":800,"shoulder":610,"bust":690,"sleeve":880}},"productType":"jacket","userGender":"male","userHeight":1750,"userWeight":68,"extProductId":"typemapperdemo.myshopify.com___products___5732631412892"}'
```

Reply:
@zr0z yes, you are right. I don't need it. I just made the change under commit 2d3a445 
In addition, I confirmed that user height should be int or float and the user weight should be float, so I changed the data types for those params as well.